src/routes/products/functions/(components)/DeploySeamlessly.svelte

```diff
@@ -0,0 +1,221 @@
+<script lang="ts">
+    import { classNames } from '$lib/utils/classnames';
+</script>
+
+<section class="pt-32 pb-40">
+    <div
+        class="container grid grid-cols-1 items-center justify-between gap-12 md:grid-cols-12 md:flex-row"
+    >
+        <div class="col-span-4 mb-10 flex flex-col">
+            <h2 class="text-title text-primary font-aeonik-pro my-4 text-pretty">
+                Deploy seamlessly with no effort
+            </h2>
+            <p class="text-body text-secondary text-pretty font-medium">
+                Appwrite offers to deploy functions directly from Git repositories to track changes
+                within your development workflow seamlessly.
+            </p>
+
+            <a href="/docs/sdks#server" class="web-button is-secondary mt-8">Learn more</a>
+        </div>
+
+        <div
+            class="window col-span-7 col-start-6 flex h-[420px] w-full flex-col bg-[#1B1B1C]/94 pb-4 backdrop-blur-lg"
```

Comment:
I see some hardcoded colours that get used quite a few times (e.g. 1B1B1C 5 times). Do you think it's worth to put it into a var?